### ISSUE TITLE
macosx installed sdl2

### ISSUE THREAD

**Issue by yiyongooi** [kivy/kivy#4936]
sorry, silly me.

i copy pasted 'USE_OSX_FRAMEWORKS=0 pip install kivy' to terminal

i should have 'env USE_OSX_FRAMEWORKS=0' and then 'pip install kivy'.

however, my next issue is SDL2. i have it installed already but when i build my project
the console still show as pygame as window. is it normal?

`[INFO              ] [Text        ] Provider: pygame
[INFO              ] [Window      ] Provider: pygame`

if i did not have pygame installed, the console will show missing pygame module.

`import kivy
from kivy.app import App
from kivy.uix.button import Label`

these are my import

**Comment by yiyongooi** [comment]
[INFO   ] [Logger      ] Record log in /Users/YIYONG/.kivy/logs/kivy_17-01-19_72.txt
[INFO   ] [Kivy        ] v1.9.2-dev0
[INFO   ] [Python      ] v2.7.10 (default, Oct 23 2015, 19:19:21) 
[GCC 4.2.1 Compatible Apple LLVM 7.0.0 (clang-700.0.59.5)]
[INFO   ] [Factory     ] 193 symbols loaded
[INFO   ] [Image       ] Providers: img_tex, img_imageio, img_dds, img_pygame, img_pil, img_gif (img_ffpyplayer ignored)
[INFO   ] [Text        ] Provider: pygame
[INFO   ] [Window      ] Provider: pygame
[CRITICAL] [Window      ] Unable to find any valuable Window provider.
pygame - TypeError: 'NoneType' object has no attribute '__getitem__'
  File "/Users/YIYONG/Library/Python/2.7/lib/python/site-packages/kivy/core/__init__.py", line 67, in core_select_lib
    cls = cls()
  File "/Users/YIYONG/Library/Python/2.7/lib/python/site-packages/kivy/core/window/__init__.py", line 817, in __init__
    super(WindowBase, self).__init__(**kwargs)
  File "kivy/_event.pyx", line 273, in kivy._event.EventDispatcher.__init__ (kivy/_event.c:5375)
    setattr(self, key, value)
  File "kivy/properties.pyx", line 423, in kivy.properties.Property.__set__ (kivy/properties.c:5171)
    self.set(obj, val)
  File "kivy/properties.pyx", line 1416, in kivy.properties.AliasProperty.set (kivy/properties.c:23280)
    if ps.setter(obj, value):
  File "/Users/YIYONG/Library/Python/2.7/lib/python/site-packages/kivy/core/window/__init__.py", line 701, in _set_top
    self._set_window_pos(pos[0], value)
[CRITICAL] [App         ] Unable to get a Window, abort.
 Exception SystemExit: 1 in 'kivy.properties.dpi2px' ignored
[CRITICAL] [App         ] Unable to get a Window, abort.


this is the current error

**Comment by FeralBytes** [comment]
Can you please give exact output to include the command that is being issued. Break up each command and it's output into different code blocks using github flavor of markdown.  Because right now I see some output about kivy gardens, and in the next a compile cython compile error, and next some brew install output which is being called the current error, and finally another compile error from setuptools.
Too hard to follow what was done when and where.
example:
```
sudo pip3 install kivy
password:
fetching python3-kivy
...... etc
```

**Comment by yiyongooi** [comment]
this is the error when i run USE_OSX_FRAMEWORKS=0 pip install kivy
`Command "/Library/Frameworks/Python.framework/Versions/3.6/bin/python3.6 -u -c "import setuptools, tokenize;__file__='/private/var/folders/vv/0wgh_t6s7311rcn9bwqcvj800000gn/T/pip-build-_rnwnzk3/kivy/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /var/folders/vv/0wgh_t6s7311rcn9bwqcvj800000gn/T/pip-eml54k9k-record/install-record.txt --single-version-externally-managed --compile" failed with error code 1 in /private/var/folders/vv/0wgh_t6s7311rcn9bwqcvj800000gn/T/pip-build-_rnwnzk3/kivy/`
